util/htmlTemplate.ts

```diff
@@ -58,14 +75,15 @@ export const htmlTemplate = async ({
     ${scripts}
   </body>`;
 
-  return `
-<!doctype html>
-<html${makeHtmlAttributes(attributes.html)}>
-  <head>
-    ${metas}
-    <title>${title}</title>
-    ${links}
-  </head>
-${body}
-</html>`;
+  return (
+    `<!doctype html>
+      <html${makeHtmlAttributes(attributes.html)}>
```

Comment:
@Soremwar was this suggested by deno fmt? because this will cause unnecessary indenting in the html template

Reply:
No, it was me

Forgot "`" also took indenting